movie_REST_API/movie_REST_API/AssemblyBuilder/AssemblyBuilder.swift

```diff
@@ -9,15 +9,26 @@ final class AssemblyBuilder: AssemblyBuilderProtocol {
 
     func createMovieModule() -> UIViewController {
         let networkService = NetworkService()
-        let imageService = ImageService()
-        let viewModel = MovieViewModel(networkService: networkService, imageService: imageService)
+        let keychainService = KeychainService()
+        let imageNetworkService = ImageNetworkService()
+        let fileService = FileService()
+        let imageService = ImageService(imageNetworkService: imageNetworkService, fileService: fileService)
```

Comment:
```suggestion
        let imageService = ImageService(proxy:  proxyService)
```
 а уже в прокс сидит imageNetworkService и fileService
 у него внутри логика откуда ему достать картинку и в какой сервис постучаться